lib/inquisition/active_record_doctor/runner.rb

```diff
@@ -30,6 +30,10 @@ def create_message(ard_task, issue_object, details)
         issue_text = ard_task.to_s.split('::').last.split(/(?=[A-Z])/).map(&:downcase).join(' ')
         "#{issue_object} has #{issue_text}, details: #{details ? details.join(', ') : 'n/a'}"
       end
+
+      def load_environment
```

Comment:
it's not DRY

Reply:
Yes, you are right, it will be fixed in the next iteration.